app/components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+// LIBRARY IMPORTS
+import { Box, Heading } from "@chakra-ui/react";
+import Image from "next/image";
+
+// LOCAL IMPORTS
+import heroImage from "../../public/images/hero-photo.png";
+
+// HERO COMPONENT DEFINITION
+export default function Hero() {
+  return (
+    <Box
+      position="relative"
+      textAlign="center"
+      bg="white"
+      minH="calc(38.25vw + 6rem)"
+    >
+      <Image
+        src={heroImage}
+        alt={"machines photo"}
+        layout="responsive"
+        width={1728}
+        height={610}
+        objectFit="cover"
+        objectPosition="center"
+      />
+      <Box
+        position="absolute"
+        w="93%"
+        left="50%"
+        transform="translate(-50%, -50%)"
+        height="6rem"
```

Comment:
This height looks a bit smaller than what's on Figma; again, I'm still a bit vague on how strictly we want to adhere to what's on Figma in terms of spacing and sizing, so disregard if it's not that important

Reply:
Great noticing, I didn't have a great way to compare the height since screen heights all vary. We should ask Kayla if she has any insight on this too.